src/Util/Config.php

```diff
@@ -45,6 +45,7 @@ public function getAttributes($config, $container = '.tiny-slider-container'): s
 
             $configData = $this->createConfig($config);
             $configData['container'] = $container;
+            $configData['language'] = $GLOBALS['TL_LANGUAGE'];
```

Comment:
irgendwie fehlt der Code, wo die Labels in die Config eingebaut werden? Oder übersehen ich was?

Reply:
die komplette dca wird in die config reingeladen, für Sachen ohne dca (z.B oben container, language, wäre dann an dieser Stelle in die Config erweitert )